core/web/migrate_controller.go

```diff
@@ -140,24 +140,31 @@ func BuildFMTaskDAG(js models.JobSpec) (string, pipeline.TaskDAG, error) {
 		// Support anyways just in case someone was using it without our knowledge.
 		// ALL fm jobs are POSTs see
 		// https://github.com/smartcontractkit/chainlink/blob/e5957895e3aa4947c2ddb5a4a8525041639962e9/core/services/fluxmonitor/fetchers.go#L67
-		var n *pipeline.TaskDAGNode
+		var httpTask *pipeline.TaskDAGNode
 		if feed.IsObject() && feed.Get("bridge").Exists() {
-			n = pipeline.NewTaskDAGNode(dg.NewNode(), fmt.Sprintf("feed%d", i), map[string]string{
+			httpTask = pipeline.NewTaskDAGNode(dg.NewNode(), fmt.Sprintf("feed%d", i), map[string]string{
 				"type":        pipeline.TaskTypeBridge.String(),
 				"method":      "POST",
 				"name":        feed.Get("bridge").String(),
 				"requestData": js.Initiators[0].InitiatorParams.RequestData.String(),
 			})
 		} else {
-			n = pipeline.NewTaskDAGNode(dg.NewNode(), fmt.Sprintf("feed%d", i), map[string]string{
+			httpTask = pipeline.NewTaskDAGNode(dg.NewNode(), fmt.Sprintf("feed%d", i), map[string]string{
 				"type":        pipeline.TaskTypeHTTP.String(),
 				"method":      "POST",
 				"url":         feed.String(),
 				"requestData": js.Initiators[0].InitiatorParams.RequestData.String(),
 			})
 		}
-		dg.AddNode(n)
-		dg.SetEdge(dg.NewEdge(n, medianTask))
+		dg.AddNode(httpTask)
+		// We always implicity parse {"data": {"result": X}}
+		parseTask := pipeline.NewTaskDAGNode(dg.NewNode(), fmt.Sprintf("jsonparse%d", i), map[string]string{
+			"type": pipeline.TaskTypeJSONParse.String(),
+			"path": "data,result",
+		})
```

Comment:
This feels extremely hacky.

Is there any way we can leave this behaviour behind with v1 instead of carrying it forwards with us?